packages/pegasus/src/pegasus.js

```diff
@@ -314,7 +313,7 @@ const makePegasus = (zcf, board, namesByAddress) => {
 
   return Far('pegasus', {
     /**
-     * Return a handler that can be used with the Network API.
+     * Return a handler that can be used with the uuuuuuuu API.
```

Comment:
intermediate refactor?

Reply:
Heheh... leaned on the keyboard at the wrong time. :)